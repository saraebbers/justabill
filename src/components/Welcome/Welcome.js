import React from 'react';
import '../../index.scss';

const Welcome = () => {

  return(
    <div className='welcome'>
      <section className='welcome-message'>
        <h3>
          Welcome Teachers and Life-long Students 
        </h3>
        <p>
          This application aggrogates infromation on bills that have become law.  Hopefully you can use this information to engage your students with current issues as you help them to understand the process of how a bill becomes law.  We invite you to save topics for later referance by clicking in the top right of the information.
        </p>
      </section>
    </div>
  )
}

export default Welcome;