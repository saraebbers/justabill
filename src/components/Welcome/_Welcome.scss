.welcome {
  background-color: #DBCFB0;
  display: grid;
  grid-template-columns: auto;
  justify-items: center;
  justify-content: space-around;
}

.welcome-message {
  border: solid #718F94 5px;
  background-color: #ADA380;
  padding: 15px;
  margin: 20px;
  width: 90%;
  color: #151E3F;

  h3 {
    text-align: center;
    font-size: 1.75rem;
    padding: 10px;
  }

  p {
    padding-bottom: 10px
  }
}

