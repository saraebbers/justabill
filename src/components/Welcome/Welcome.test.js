import React from 'react';
import Welcome from './Welcome';
import { shallow } from 'enzyme';

describe('Welcome', () => {
  it('it should match the snapshot',() => {
  const wrapper = shallow(<Welcome />)
  expect(wrapper).toMatchSnapshot()
  })
})