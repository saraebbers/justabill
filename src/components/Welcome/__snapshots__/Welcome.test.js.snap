// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome it should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Welcome />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <section
        className="welcome-message"
      >
        <h3>
          Welcome Teachers and Life-long Students
        </h3>
        <p>
          This application aggrogates infromation on bills that have become law.  Hopefully you can use this information to engage your students with current issues as you help them to understand the process of how a bill becomes law.  We invite you to save topics for later reference by clicking in the top right of the information.
        </p>
      </section>,
      "className": "welcome",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h3>
            Welcome Teachers and Life-long Students
          </h3>,
          <p>
            This application aggrogates infromation on bills that have become law.  Hopefully you can use this information to engage your students with current issues as you help them to understand the process of how a bill becomes law.  We invite you to save topics for later reference by clicking in the top right of the information.
          </p>,
        ],
        "className": "welcome-message",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Welcome Teachers and Life-long Students",
          },
          "ref": null,
          "rendered": "Welcome Teachers and Life-long Students",
          "type": "h3",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "This application aggrogates infromation on bills that have become law.  Hopefully you can use this information to engage your students with current issues as you help them to understand the process of how a bill becomes law.  We invite you to save topics for later reference by clicking in the top right of the information.",
          },
          "ref": null,
          "rendered": "This application aggrogates infromation on bills that have become law.  Hopefully you can use this information to engage your students with current issues as you help them to understand the process of how a bill becomes law.  We invite you to save topics for later reference by clicking in the top right of the information.",
          "type": "p",
        },
      ],
      "type": "section",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <section
          className="welcome-message"
        >
          <h3>
            Welcome Teachers and Life-long Students
          </h3>
          <p>
            This application aggrogates infromation on bills that have become law.  Hopefully you can use this information to engage your students with current issues as you help them to understand the process of how a bill becomes law.  We invite you to save topics for later reference by clicking in the top right of the information.
          </p>
        </section>,
        "className": "welcome",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h3>
              Welcome Teachers and Life-long Students
            </h3>,
            <p>
              This application aggrogates infromation on bills that have become law.  Hopefully you can use this information to engage your students with current issues as you help them to understand the process of how a bill becomes law.  We invite you to save topics for later reference by clicking in the top right of the information.
            </p>,
          ],
          "className": "welcome-message",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Welcome Teachers and Life-long Students",
            },
            "ref": null,
            "rendered": "Welcome Teachers and Life-long Students",
            "type": "h3",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "This application aggrogates infromation on bills that have become law.  Hopefully you can use this information to engage your students with current issues as you help them to understand the process of how a bill becomes law.  We invite you to save topics for later reference by clicking in the top right of the information.",
            },
            "ref": null,
            "rendered": "This application aggrogates infromation on bills that have become law.  Hopefully you can use this information to engage your students with current issues as you help them to understand the process of how a bill becomes law.  We invite you to save topics for later reference by clicking in the top right of the information.",
            "type": "p",
          },
        ],
        "type": "section",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
